front/app/services/permissions/rules/routePermissions.ts

```diff
@@ -66,6 +66,10 @@ export const canAccessRoute = (
       return true;
     }
 
+    if (item.path.includes('folders')) {
```

Comment:
Obviously not a good fix at all, but brings the app back to the previous functionality.

Reply:
Leaving it in the main app should be fine for now, as it returns false if the user is doing anything other than folders-related, so should only be a narrow fix for this part of the app.